packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -43,6 +45,35 @@ export function isAPISupportedByPlatform(requiredVersion: string = defaultSDKVer
   return value >= 0;
 }
 
+/**
+ * @hidden
+ * Helper function to identify if host client is either android or ios
+ *
+ * @internal
+ */
+export function isHostClientMobile(): boolean {
+  return GlobalVars.hostClientType == HostClientType.android || GlobalVars.hostClientType == HostClientType.ios;
+}
+
+/**
+ * @hidden
+ * Helper function which indicates if current API is supported on mobile or not.
+ * @returns SdkError if host client is not android/ios or if the requiredVersion is not
+ *          supported by platform or not. Null is returned in case of success.
+ *
+ * @internal
+ */
+export function isApiSupportedOnMobile(requiredVersion: string = defaultSDKVersionForCompatCheck): SdkError {
```

Comment:
Why are we returning an error? Isn't it standard practice to throw an Error when it is not supported?

Reply:
I'm not entirely sure, but I think they're currently setting up for more changes in this area so I'm thinking it might be best to leave it be since they might have reasons for building their helper in this specific way and it's also internal so we're not expecting anyone else to use it.

Reply:
I agree with Nikahira, either throwing an error or returning a boolean would make more sense. I am guessing the PR on the main branch contains this change? I am wondering if a PR on the main branch would be appropriate?

Reply:
I also agree with Nikahira. It should probably either throw an error or return a boolean. I am wondering if a PR on the main branch would be appropriate to fix this?

Reply:
I would change it to throw Error. (Do it right the first time). We can always change future releases (dependent functions) to have it called the right way.

Reply:
In that case, I think I should reach out to the v1 folks before this is merged so it's consistent across the versions.

Reply:
To get unblocked, tabling this issue. Will create a work item.

Reply:
@aeunms can you please reply to this and link the bug or work item you open for it?

In order to unblock the merge, we're going to have whoever the current git magician is focus on opening bugs for addressing any complicated changes that come up during merge PRs. We'll let whoever picks those bugs up figure out whether or not the change makes sense in v1 or v2 (v1 seems sensible here)

Reply:
As discussed offline, I am creating a bug item and linking it in our Teams channel.